core/src/test/java/org/jdbi/v3/core/SqlTest.java

```diff
@@ -29,13 +30,20 @@ public class SqlTest {
 
     @Test
     void testEmptyInput() {
-        assertEquals("", Sql.of((CharSequence[]) null).toString());
+        assertEquals("", Sql.of((CharSequence) null).toString());
         assertEquals("", Sql.of((Collection<CharSequence>) null).toString());
         assertEquals("", Sql.of(new CharSequence[0]).toString());
         assertEquals("", Sql.of(new ArrayList<>()).toString());
         assertEquals("", Sql.of().toString());
     }
 
+    @Test
+    void testMultipleStrings() {
+        assertEquals("A B C", Sql.of(ImmutableList.of("A", "B", "C")).toString());
```

Comment:
I would recommend the JDK's `List.of` which is fully immutable (including set operations). This would prevent the usage of Guava in this test class.

Reply:
Correct. List.of requires JDK9. ;-)